webrtc.html

```diff
@@ -2641,9 +2641,10 @@
           <dd>
             <p>The <code>errorCode</code> attribute is the numeric STUN
             error code returned by the STUN or TURN server [[STUN-PARAMETERS]].</p>
-            <p>If the server could not be reached, <code>errorCode</code> will
+            <p>If the server could not be reached from all host candidates, <code>errorCode</code> will
```

Comment:
I think this would be more clear if worded as "If no host candidate could reach the server, ..."